sample/Ookii.Dialogs.WinForms.Sample/Ookii.Dialogs.WinForms.Sample.csproj

```diff
@@ -1,41 +1,27 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{D52F3B1C-03B3-41B1-A15E-E23FF6877939}</ProjectGuid>
     <OutputType>WinExe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Ookii.Dialogs.WinForms.Sample</RootNamespace>
-    <AssemblyName>Ookii.Dialogs.WinForms.Sample</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <!--<TargetFramework>net45</TargetFramework>-->
+    <!--<TargetFramework>netcoreapp3.1</TargetFramework>-->
+    <TargetFramework>net5.0-windows</TargetFramework>
     <ApplicationManifest>app.manifest</ApplicationManifest>
     <ApplicationIcon>ookii.ico</ApplicationIcon>
-    <TargetFrameworkProfile />
+    <AssemblyTitle>Ookii.Dialogs.WinForms.Sample</AssemblyTitle>
+    <Company>augustoproiete.net</Company>
+    <Product>Ookii.Dialogs.WinForms</Product>
+    <Copyright>Copyright (c) 2009-2021 Ookii Dialogs Contributors</Copyright>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <Version>1.1.0</Version>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
```

Comment:
I think we can exclude this line and leave the default that MSBuild generates

Reply:
Defaulted to 1.0.0.